backend/nodes/image_dim_nodes.py

```diff
@@ -194,3 +194,36 @@ def run(
         result = img[top : h - bottom, left : w - right]
 
         return result
+
+
+@NodeFactory.register("chainner:image:get_dims")
+class GetDimensionsNode(NodeBase):
+    """Node for getting the dimensions of an image"""
+
+    def __init__(self):
+        """Constructor"""
+        super().__init__()
+        self.description = (
+            "Get the Height, Width, and number of Channels from an image."
+        )
+        self.inputs = [
+            ImageInput(),
+        ]
+        self.outputs = [
+            NumberOutput("Height"),
+            NumberOutput("Width"),
+            NumberOutput("Channels"),
+        ]
+        self.category = IMAGE_DIMENSION
+        self.name = "Get Dimensions"
+        self.icon = "BsRulers"
+        self.sub = "Utility"
+
+    def run(
+        self,
+        img: np.ndarray,
+    ) -> any:
```

Comment:
Why an `any` type hint? 

I don't know anything about py type hints, but wouldn't this stop the IDE from inferring the return type?

Reply:
Oh yeah, I forgot to finish that